README.md

```diff
@@ -14,11 +14,143 @@
+   * [(most) Permament Install](#most-permanent-install)
+* [The Fine Print](#the-fine-print)
+* [Getting Help](#getting-help)
+
+## Developing
+
+### Requirements
+
+* [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
+* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+* Network access to https://repository.sonatype.org/content/groups/sonatype-public-grid
+
+Also, there is a good amount of information available at [Bundle Development](https://help.sonatype.com/display/NXRM3/Bundle+Development)
+
+### Building
+
+To build the project and generate the bundle use Maven
+
+    mvn clean package
+
+If everything checks out, the bundle for chef should be available in the `target` folder
+
+#### Build with Docker
+
+`docker build -t nexus-repository-chef:0.0.1 .`
+
+#### Run as a Docker container
+
+`docker run -d -p 8081:8081 --name nexus nexus-repository-chef:0.0.1` 
+
+For further information like how to persist volumes check out [the GitHub repo for our official image](https://github.com/sonatype/docker-nexus3).
+
+The application will now be available from your browser at http://localhost:8081
+
+## Using Chef With Nexus Repository Manager 3
+
+[We have detailed instructions on how to get started here!](docs/CHEF_USER_DOCUMENTATION.md)
+
+## Compatibility with Nexus Repository Manager 3 Versions
+
+The table below outlines what version of Nexus Repository the plugin was built against
+
+| Plugin Version | Nexus Repository Version |
+|----------------|--------------------------|
+| v0.0.1         | 3.13.0-01                |
+
+If a new version of Nexus Repository is released and the plugin needs changes, a new release will be made, and this
+table will be updated to indicate which version of Nexus Repository it will function against. This is done on a time 
+available basis, as this is community supported. If you see a new version of Nexus Repository, go ahead and update the
+plugin and send us a PR after testing it out!
+
+All released versions can be found [here](https://github.com/sonatype-nexus-community/nexus-repository-chef/releases).
+
+## Features Implemented In This Plugin 
+
+| Feature | Implemented          |
+|---------|----------------------|
+| Proxy   | :heavy_check_mark: * |
+| Hosted  |                      |
+| Group   |                      |
+
+`* tested primarily against the Chef Supermarket, not guaranteed to work on the wide wild world of Chef repositories.`
+
+## Installing the plugin
+
+There are a range of options for installing the chef plugin. You'll need to build it first, and
+then install the plugin with the options shown below:
+
+### Temporary Install
+
+Installations done via the Karaf console will be wiped out with every restart of Nexus Repository. This is a
+good installation path if you are just testing or doing development on the plugin.
+
+* Enable Nexus console: edit `<nexus_dir>/bin/nexus.vmoptions` and change `karaf.startLocalConsole`  to `true`.
```

Comment:
Minor: While I suspect this is just copypasta from template (as well as a community perception that we fight to correct), believe "Nexus" here should be "NXRM" (or something akin).

Reply:
In: 507010a